src/sentry/api/endpoints/api_tokens.py

```diff
@@ -35,11 +37,29 @@ class ApiTokensEndpoint(Endpoint):
     authentication_classes = (SessionNoAuthTokenAuthentication,)
     permission_classes = (IsAuthenticated,)
 
-    @method_decorator(never_cache)
-    def get(self, request: Request) -> Response:
+    @classmethod
+    def _get_appropriate_user_id(cls, request: Request) -> str:
+        """
+        Gets the user id to use for the request, based on what the current state of the request is.
+        If the request is made by a superuser, then they are allowed to act on behalf of other user's data.
+        Therefore, when GET or DELETE endpoints are invoked by the superuser, they can use another user id.
```

Comment:
> they can use another user id.
a little confusing. can we change this to something like `we may utilize a provided user_id` or something?

Reply:
good suggestion, let me update!

Reply:
done!